README.md

```diff
@@ -66,12 +66,12 @@ In this way evildoers can't reuse a sign-up form's SGID on the login page. For e
 => #<Person:0x007fae94bf6298 @id="1">
 ```
 
-You can also have SGIDs that expire some time in the future. This is useful if there's a resource,
-people shouldn't have indefinite access to, like a share link.
+By default, SGIDs expire after a month. Specify the lifetime of an SGID when
+generating it using the `:expires_in` and `:expires_at` options.
```

Comment:
I'm sad that we're booting the flow of the original text here. Can't we integrate the default 1 month expiry into it?